packages/modern-test-app/package.json

```diff
@@ -63,7 +63,7 @@
     "ember-source-channel-url": "^3.0.0",
     "ember-template-imports": "^3.4.2",
     "ember-template-lint": "^5.11.0",
-    "ember-truth-helpers": "4.0.1",
+    "ember-truth-helpers": "workspace:*",
```

Comment:
@NullVoxPopuli then you should update https://github.com/jmurphyau/ember-truth-helpers/blob/v4.0.1/package.json#L48-L49

btw is there some issue this fixes? maybe it's only me, but I haven't came across problems (yet?) using version instead of workspace protocol

Reply:
why does release-it need to care about workspaces? if it's ran  from the addon directory, it can function as is

Reply:
@NullVoxPopuli currently it's setup to run in the monorepo root

Reply:
I remove the updating config.

The main thing is that we need to be certain that we're using the workspace local copy, and not the "last published".

Reply:
> The main thing is that we need to be certain that we're using the workspace local copy, and not the "last published".

so it's more like a safety net in case if for whatever reason in the universe version in `package.json` gets out of sync with the actual version in the monorepo rather than fix for some existing error?

Reply:
it looks like the important parts remain good:
![image](https://github.com/jmurphyau/ember-truth-helpers/assets/199018/75ac51be-1dd6-4167-94d9-54b23c655e39)

Reply:
> with the actual version in the monorepo rather than fix for some existing error?

yeah, it's happened before, depending on `.npmrc` changes and minor pnpm releases

Reply:
@NullVoxPopuli got it! Thanks for explanation so I better understand!